core-primitives/types/src/parentchain.rs

```diff
@@ -52,6 +55,18 @@ pub enum ParentchainId {
 	TargetB,
 }
 
+#[cfg(feature = "std")]
+impl Display for ParentchainId {
+	fn fmt(&self, f: &mut std::fmt::Formatter) -> std::fmt::Result {
+		let message = match self {
+			ParentchainId::Integritee => "L1:Integritee",
+			ParentchainId::TargetA => "L1:Encointer",
+			ParentchainId::TargetB => "L1:NodeTemplate",
+		};
+		write!(f, "{}", message)
+	}
+}
+
```

Comment:
I'd like this to be upstreamed and customizeable. but it should work inside enclave too and should replace all log prefixes

Reply:
Display is part of `core`, so it can work inside the enclave too.

I am unsure if it makes sense to replace the general log prefixes. It is not the same as in the substrate runtime, where we know that the `encointer` logger target is always in the runtime. Here it can be any crate in a fairly complex setup, so we should keep the crate path in in the log IMO.

Reply:
pardon, I didn't mean the pallet log prefixes. I meant the ones where we deliberately printed TargetA/B/Integritee before.
crate path in the log is fine, but the parentchain should become part of the prefix or be therwise prepended to the log message  where it makes sense